### DISCUSSION TITLE
Is it possible to combine lockFileMaintenance and package updates together in the same PR?

### DISCUSSION THREAD

**Discussion by paulzakin** [renovatebot/renovate#19691] [Request Help] [answered]
Let's say I have this `renovate.json`.  Is it possible to combine lockFileMaintenance and the one entry into package rules together into one PR?

```json
{
  "extends": ["config:base", "docker:enableMajor"],
  "labels": ["dependency"],
  "configMigration": true,
  "packageRules": [
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["minor", "patch"],
      "groupSlug": "all-minor-patch",
      "groupName": "all non-major dependencies",
      "schedule": ["before 6:00am on Monday"],
      "automerge": true
    },
  ],
  "lockFileMaintenance": {
    "enabled": true,
    "automerge": true,
    "schedule": ["before 6:00am on Tuesday"]
  }
}

```

**Comment by viceice** [comment, marked as answer]
it's not supported but maybe works

**Reply by paulzakin** [reply]
Got it, thanks!

**Reply by nejch** [reply]
@paulzakin did you end up trying anything that happens to work?

I tried a quick attempt at force-grouping it with the other updateTypes but no luck, haven't dug more into it though.

**Reply by paulzakin** [reply]
@nejch Nope! Right now it is only a minor annoyance (two PR vs. one PR) each week. Eventually something I may look into though
